src/content/code/plugin-api/camera/camera-viewport.ts

```
@@ -0,0 +1,32 @@
+// @ts-nocheck
+
+// Example 1: Access and log the current geographic boundaries of the viewport
+const viewportBounds = reearth.camera.viewport;
+
+if (viewportBounds) {
```

Biome

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)